config/sync/core.entity_view_display.node.stanford_page.default.yml

```diff
@@ -27,6 +27,7 @@ third_party_settings:
         layout_settings:
           extra_classes: ''
           centered: null
+          columns: default
```

Comment:
This isn't really part of this update but good to have. I know what this is about.